src/mixins/filing-template-mixin.ts

```diff
@@ -496,7 +496,7 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
     const shareStructure = filing.alteration.shareStructure
     this.setShareClasses(shareStructure.shareClasses)
     this.setResolutionDates(shareStructure.resolutionDates)
-    this.setOriginalResolutionDates(this.getSnapshotShareStructure?.resolutionDates || null)
+    this.setOriginalResolutionDates(businessSnapshot.resolutions)
```

Comment:
Should this fall back to "null" if it's undefined, or is it never undefined?

Reply:
Well it could be an empty [] but i don't think that's a problem right? 
If the request fails, we never make it into the failing i'm pretty sure.

Reply:
Empty array is fine. I'm wondering if we would ever do `resolutions[0].date` (which would fail if resolutions is null.

As long as the API returns [] when there are no resolutions (re: fetchResolutions()) then it should be fine.

(What I'm checking: initial state, whatever we might assign to it, and how we might use it.)

TLDR: confirm API and if OK then nothing needs to be done here.

Reply:
Can Confirm 👍 

![Screen Shot 2021-04-20 at 7 04 53 AM](https://user-images.githubusercontent.com/53542131/115434799-f5395c00-a1bd-11eb-85bc-6f860a07892e.png)